packages/flutter_tools/lib/src/run.dart

```diff
@@ -31,7 +31,7 @@ class RunAndStayResident extends ResidentRunner {
   String _mainPath;
   LaunchResult _result;
 
-  /// Start the app and keep the process running during its lifetime.
+  @override
   Future<int> run({
     bool traceStartup: false,
```

Comment:
Can these parameters be passed into the RunAndStayResidentRunner's constructor so that the run interface is the same in both subclasses?